src/modules/Home/Home.js

```diff
@@ -1,23 +1,59 @@
 import React from 'react';
-import AppLogo from '../../assets/applogo.svg';
+import {useState} from 'react';
+import './Home.css';
 
 function Home() {
+  const API_KEY = process.env.REACT_APP_NAPSTER_API_KEY;
+  const [genreData, setGenreData] = useState([]);
+  if (!genreData.length) {
+    console.log(genreData)
+    let genres = [];
+    //fetch genre images.
+    fetch('https://api.napster.com/v2.2/genres?apikey=' + API_KEY)
+    .then(function(response) {
+      // Successful response :)
+      console.log("success");
+      return response.json(); 
+    })
+    .then(function(response) {
+      console.log(response)
+      response.genres.forEach(genre => {
+        //don't need to api call the image?
+        genres.push({
+          genreName: genre.name,
+          genreImageSrc: "https://api.napster.com/imageserver/images/" + genre.id + "/240x160.jpg"
+        })
+      })
+      console.log(genres);
+      setGenreData(genres);
+    })
+    .catch(function(err) {
+      // Error :(
+        console.log("Error: unable to fetch genre data");
+    });
+  }
+  let genreDOMEles = [];
```

Comment:
What does "DOMEles" mean?

Reply:
Yeah. What does "DOMEles" mean? Please explain your variable name usage.

Reply:
Short for DOMElements. It is an array that holds the pseudo-HTML elements to be rendered in the DOM hence the name.